int8 direction_x
int8 direction_y
int8 wheel_rotation

bool btn_right
bool btn_left
bool btn_middle
